### ISSUE TITLE
Pagekite configuration page throws 500 error

### ISSUE THREAD

**Issue by JosephKiranBabu** [freedombox/Plinth#900]
Tried to go to the "Configure PageKite" tab after installing it. Got the following error.

```
[2017-07-12 11:55:51,707] plinth.actions INFO     Executing command - ['sudo', '-n', '/usr/share/plinth/actions/pagekite', 'is-disabled']
[2017-07-12 11:55:55,186] plinth.actions INFO     Executing command - ['sudo', '-n', '/usr/share/plinth/actions/pagekite', 'get-kite']
[2017-07-12 11:55:58,571] plinth.actions INFO     Executing command - ['sudo', '-n', '/usr/share/plinth/actions/pagekite', 'get-frontend']
[2017-07-12 11:56:02,451] django.request ERROR    Internal Server Error: /plinth/sys/pagekite/configure/
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/usr/lib/python3/dist-packages/django/core/handlers/base.py", line 249, in _legacy_get_response
    response = self._get_response(request)
  File "/usr/lib/python3/dist-packages/django/core/handlers/base.py", line 217, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/usr/lib/python3/dist-packages/django/core/handlers/base.py", line 215, in _get_response
    response = response.render()
  File "/usr/lib/python3/dist-packages/django/template/response.py", line 107, in render
    self.content = self.rendered_content
  File "/usr/lib/python3/dist-packages/django/template/response.py", line 84, in rendered_content
    content = template.render(context, self._request)
  File "/usr/lib/python3/dist-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib/python3/dist-packages/django/template/loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib/python3/dist-packages/django/template/loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib/python3/dist-packages/django/template/loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 1040, in render
    output = self.filter_expression.resolve(context)
  File "/usr/lib/python3/dist-packages/django/template/base.py", line 736, in resolve
    new_obj = func(obj, *arg_vals)
  File "/usr/lib/python3/dist-packages/bootstrapform/templatetags/bootstrap.py", line 46, in bootstrap_horizontal
    return render(element, markup_classes)
  File "/usr/lib/python3/dist-packages/bootstrapform/templatetags/bootstrap.py", line 83, in render
    return template.render(context)
  File "/usr/lib/python3/dist-packages/django/template/backends/django.py", line 64, in render
    context = make_context(context, request, autoescape=self.backend.engine.autoescape)
  File "/usr/lib/python3/dist-packages/django/template/context.py", line 287, in make_context
    raise TypeError('context must be a dict rather than %s.' % context.__class__.__name__)
TypeError: context must be a dict rather than Context.
```

**Comment by JoKeyser** [comment]
This seems related to our own #894 and in general https://github.com/tzangms/django-bootstrap-form/issues/99. I'm puzzled why here it's an error and in #894 it's just a warning, though.
My guess is that @SunilMohanAdapa 's PR https://github.com/tzangms/django-bootstrap-form/pull/97 would fix them. I'm not 100% sure whether all this is true, please verify.
@jvalleroy , should this also be part of our [0.15 milestone](https://github.com/freedombox/Plinth/milestone/9)?
Overall I'm getting nervous about the current non-responsiveness of @tzangms, but again, I'm new to all of this; are there other good options in Debian to use bootstrap forms in Django?

**Comment by JoKeyser** [comment]
Just confirmed that Sunil's PR would solve it, by editing `/usr/lib/python3/dist-packages/bootstrapform/templatetags/bootstrap.py` and changing the 3 occurrences of `context = Context({stuff})` into `context = {stuff}`.

**Comment by JosephKiranBabu** [comment]
Closing this issue since Sunil's PR got merged into upstream.

**Comment by jvalleroy** [comment]
This issue still happens in 0.15.2.

**Comment by JosephKiranBabu** [comment]
Version 3.3 of django-bootstrap-form hasn't been packaged for Debian yet.

**Comment by JosephKiranBabu** [comment]
Debian bug number for tracking. 
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=876974

**Comment by JosephKiranBabu** [comment]
Bug fixes:
https://anonscm.debian.org/cgit/freedombox/python-django-bootstrap-form.git/log/?h=debian/unstable

**Comment by JosephKiranBabu** [comment]
Fixed in python-django-bootstrap-form 3.3-1
https://tracker.debian.org/news/894656
